core/src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -560,7 +561,7 @@ public void execute(Object player, String[] args) {
     public TabCompleter getTabCompleter() {
         MethodInterface mi = Universal.get().getMethods();
         if (!mi.getBoolean(mi.getConfig(), "Use Tab Completion", true)) 
-            return new NullTabCompleter();
+            return nullTabCompleter;
```

Comment:
Sorry, if you could just change the name from _nullTabCompleter_ to _NULL_TAB_COMPLETER_ as it is a constant. Then we will be ready to merge :)

Reply:
Done!